src/main/java/org/akhq/configs/Ldap.java

```diff
@@ -0,0 +1,56 @@
+package org.akhq.configs;
+
+import io.micronaut.context.annotation.ConfigurationProperties;
+import io.micronaut.core.util.StringUtils;
+import lombok.Data;
+import org.akhq.utils.UserGroupUtils;
+
+import javax.annotation.PostConstruct;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@ConfigurationProperties("akhq.security.ldap")
+@Data
+public class Ldap {
+    private String defaultGroup;
+    private Map<String, GroupMapping> groups = new HashMap<>();
```

Comment:
There is a lot of issue using a Map<String, T>, micronaut change the case of key and user complain about that.
I think we must go with a List<T> here  as `group`

Reply:
Yeah, I only added it this way to stay consistent with the other entries. If we change it here, I'll change it in all occurences (oidc, ldap, basic auth, general groups)

Reply:
There is a weird bug with `@EachProperty` in list mode. If you use it, it will not fill List<String> fields below it. I had to remove any `@EachProperty` and use a new `SecurityProperties` class for it.